lib/tapioca/generators/init.rb

```diff
@@ -0,0 +1,84 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Tapioca
+  module Generators
+    class Init < Base
+      sig do
+        params(
+          file_writer: Thor::Actions,
```

Comment:
Isn't `Thor::Actions` a [module](https://rubydoc.info/github/wycats/thor/master/Thor/Actions)? Why can't we include it instead of using composition here?

Reply:
Yeah, we discussed this in the Slack thread. But, basically, it overrides the `initialize` method and injects `ClassMethods` into the class. So composition, in this case, is the better solution. As @paracycle notes below, decoupling this behaviour can be addressed in a separate PR.